tests/resources/test_powerpacks.py

```diff
@@ -86,13 +86,37 @@ def test_add_number(self):
             'https://api.plivo.com/v1/Account/MAXXXXXXXXXXXXXXXXXX/NumberPool/ca5fd1f2-26c0-43e9-a7e4-0dc426e9dd2f/Number/15799140336/',
             self.client.current_request.url)
 
+    @with_response(200)
+    def test_add_tollfree(self):
+        powerpack = self.client.powerpacks.get(uuid='d35f2e82-d387-427f-8594-6fa07613c43a')
+        response = powerpack.add_number( tollfree='18772209942')
+        #response= powerpack.numberpool.numbers.add( number='18772209942')
+        self.assertEqual('POST', self.client.current_request.method)
+        self.assertUrlEqual(
+            'https://api.plivo.com/v1/Account/MAXXXXXXXXXXXXXXXXXX/NumberPool/ca5fd1f2-26c0-43e9-a7e4-0dc426e9dd2f/Tollfree/18772209942/',
+            self.client.current_request.url)
+
     @with_response(200)
     def test_remove_number(self):
         powerpack = self.client.powerpacks.get(uuid='d35f2e82-d387-427f-8594-6fa07613c43a')
         response= powerpack.remove_number( number='15799140336')
         #response= powerpack.numberpool.numbers.remove( number='15799140336')
         self.assertEqual('DELETE', self.client.current_request.method)
 
+    @with_response(200)
```

Comment:
on remove status code is 200?

Reply:
Yes for remove number/tollfree/shortcode apis return 200